R/asNumericIfPossible.R

```diff
@@ -1,4 +1,8 @@
 asNumericIfPossible <- function(x, maintain.factor.scores = TRUE, force.string = TRUE, transform.factors = TRUE, varName = NULL) {
+  checkmate::assert_logical(maintain.factor.scores, len = 1)
+  checkmate::assert_logical(force.string, len = 1)
+  checkmate::assert_logical(transform.factors, len = 1)
+  #checkmate::assert_character(varName)
```

Comment:
maybe combine assert_character and assert_null?

Reply:
title of this issue is "checking varName only if specified" -- have I overlooked where varName is checked now? can't find any checks?

Reply:
I had written another check, but apparently didn't upload it correctly.